src/App.js

```diff
@@ -139,7 +139,7 @@ export default function App() {
         contrastText: "#ffffff",
       },
       secondary: {
-        main: "#ffefb7",
+        main: "#f6f3f0",
```

Comment:
not sure
```suggestion
        main: "#f2e9e4",
```

Reply:
@alexfauquette actually I first took screenshot of the website and applied colour to hex code converter to check the colour of top bar to be #f6f3f0. For further confirmation, if this colour has been used, I went through openfoodfacts-server repo and got this:

![Screenshot_2022-11-28-07-06-29-115_com.github.android.jpg](https://user-images.githubusercontent.com/80174214/204173839-14c234e2-fa7a-4f99-9d1c-55acabf9f5ac.jpg)

The `$latte` variable is used which is #f6f3f0